src/elements/content-sidebar/additional-tabs/AdditionalTab.js

```diff
@@ -35,8 +39,14 @@ class AdditionalTab extends React.PureComponent<Props, State> {
 
     render() {
         const { callback: callbackFn, id, isLoading, iconUrl, onImageLoad, title, ...rest } = this.props;
+        const { status } = rest;
+        const blockedByShieldAccessPolicy = status === 'BLOCKED_BY_SHIELD_ACCESS_POLICY';
```

Comment:
I would move a couple of these lines to its own method, so that can be re-used later for future statuses and also tested on its own. Also keeps the render function a bit smaller.

```jsx
const BLOCKED_BY_SHEILD = 'BLOCKED_BY_SHIELD_ACCESS_POLICY';

isDisabled(): boolean {
    const { status } = this.props;
    return blockedByShieldAccessPolicy = status === BLOCKED_BY_SHEILD;
}

getDisabledReason(): React.Node {
    let reason = '';
    const { status } = this.props;
    switch (status) {
        case BLOCKED_BY_SHEILD:
                reason = <FormattedMessage {...messages.blockedByShieldAccessPolicy} />;
                break;
        default:
                // noop
    }
    return reason;
}

render() {
    const isDisabled = this.isDisabled();
    const tooltipText = isDisabled ? this.getDisabledReason() : title;
}
```